bpf/bpf_overlay.c

```diff
@@ -199,7 +199,7 @@ static inline int handle_ipv4(struct __sk_buff *skb)
 #endif
 }
 
-__section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_IPV4_FROM_LXC) int tail_handle_ipv4(struct __sk_buff *skb)
+__section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_IPV4_FROM_OVERLAY) int tail_handle_ipv4(struct __sk_buff *skb)
```

Comment:
Where is this defined?

Reply:
yeah popped this off for now will think a bit about it.